<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageJDR</class>
 <widget class="QWizardPage" name="WizardPageJDR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Page 2</string>
  </property>
  <attribute name="pageId">
   <string notr="true">Page 2</string>
  </attribute>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Langue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="w_langue">
       <item>
        <property name="text">
         <string>VF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VO</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Achat</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="w_oui">
          <property name="text">
           <string>Oui</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="w_encours">
          <property name="text">
           <string>En Cours</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="w_non">
          <property name="text">
           <string>Non</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
